### PR TITLE
Fix bugs of pipeline on ascend.

### PR DESCRIPTION
**Pull request by gongweibao**
### PR types
Bug fixes

### PR changes
Others

### Describe
Fix macro's bug of pipeline on ascend platform.

### CODE REVIEW

**Review by sandyhouse — DISMISSED** [review]
LGTM

**Review by chalsliu — APPROVED** [review]

File: python/paddle/fluid/framework.py [hunk 1]
```diff
@@ -6124,9 +6124,9 @@ def device_guard(device=None):
         device, index = device.split(':')
         if device == 'cpu':
             raise ValueError("Should not set device id for cpu.")
-    if device not in ['cpu', 'gpu', '', None]:
+    if device not in ['cpu', 'gpu', 'npu', '', None]:
         raise ValueError(
-            "The Attr(device) should be 'cpu' or 'gpu', and it can also be empty string or None "
+            "The Attr(device) should be 'cpu' 'npu' or 'gpu', and it can also be empty string or None "
```
Comment (chenwhql) [inline python/paddle/fluid/framework.py:6129]: 'cpu' 'npu'，这里中间是不是应该有个逗号

**Review by chenwhql — APPROVED** [review]
LGTM for framework.py

File: python/paddle/fluid/framework.py (hunk 1 above)
Comment (gongweibao) [reply python/paddle/fluid/framework.py:6129]: Thanks. I will add it later.

**Pull request merged**